source/mig.md

```diff
@@ -0,0 +1,76 @@
+# Multi-Instance GPU (MIG)
+
+Multi-Instance GPU is a technology that allows partitioning a single GPU into multiple instances, making each one seem as a completely independent GPU. Each instance then receives a certain slice of the GPU computational resources and a pre-defined block of memory that is detached from the other instances by on-chip protections.
```

Comment:
Is there a MIG product page we could like "Multi-Instance GPU" to?

Reply:
Done in https://github.com/rapidsai/deployment/pull/63/commits/19c6a4169bc7ae9a262ca2188a82dedf0e3f0ff4 .